.github/workflows/ci.yml

```diff
@@ -7,8 +7,8 @@ on:
       - 'docs'
 
 env:
-  BUILDER_VERSION: v0.9.35
-  BUILDER_SOURCE: releases
+  BUILDER_VERSION: setup_crt_ci_from_action
+  BUILDER_SOURCE: channels
```

Comment:
Don't forget to set this back to `releases` before merging

Reply:
Thanks for the reminder and will do!